pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -901,7 +901,9 @@ func compareACE(ACE0 types.ACE, ACE1 types.ACE) bool {
 // for this, we will match either the target port or
 // the ingress protocol/lport being same
 func matchACLForPortMap(ACLs []types.ACE) bool {
-	matchTypes := []string{"protocol", "lport"}
```

Comment:
The original code was fine. Sorry for causing confusion with my confusion between lport and fport.

Reply:
reverted.